### DISCUSSION TITLE
Is it possible to disable cache for input in config file? Or pass --no-cache option to a specific task in config file?

### DISCUSSION THREAD

**Discussion by christaikobo** [Flexget/Flexget#4010] [Q&A] [answered]
I want to update a certain input source with higher frequency, but it seems that flexget caches result for 5 minutes so no actual query is sent out. Is it possible to disable this cache, or maybe pass --no-cache option through config file for specific task?

Thanks for any help!

**Comment by christaikobo** [comment, marked as answer]
It is impossible. I will have to run cron job and execute task myself, sad. 
https://github.com/Flexget/Flexget/issues/2667
Honestly, anyone who want to misuse it can easily do so, I don't see the point prohibiting the use of nocache in config.

**Reply by BrutuZ** [reply]
The friction is a hint and enough to discourage misuse most of the time.

You're right that anyone who really wants will find a way around it anyway, whether it be with the inconvenience of cron or changing how the program works since it's open source after all.

**Reply by christaikobo** [reply]
> The friction is a hint and enough to discourage misuse most of the time.

I think the friction is enough to discourage "use" all together. There is no reason why it dissuade misuse more than legit use.

**Comment by BrutuZ** [comment]
It's meant to be used manually, like on test runs during config adjustments, not in "production", so the execution flag is perfectly fit for its intended use.
